### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by notDavid** [naveednajam/Little-Snitch---Rule-GroupsV2#3]
Hi there, not sure how you are filtering domains, but why for example is `mozilla.org` being blocked in https://github.com/naveednajam/Little-Snitch---Rule-GroupsV2/blob/dd2f3bb20f55903e17372b51118008ebdd6699ef/sources/ads/adguard.english/domains#L7492

Is this a bug?

**Comment by naveednajam** [comment]
Hi David, in which list it is? I don’t do much of filtering mostly i convert the published lists into little snitch format. Some of the lists have many false positives like easylist i do try to filter out top level domains but where is relates to specific domain concerns should be raised to list maintainer. 

Or another way is to maintain a white list but it will require community effort to identify which domains should much go into white list.

**Comment by notDavid** [comment]
Hi, I have linked to the exact list and the item in my first post.

`mozilla.org` is not being blocked in the source list. It only blocks some small script inside the `mozilla.org` domain, but of course not the entire domain.

I think the problem is you are converting an `Adblock Plus filter syntax` list, to a host list, which is not possible.

**Comment by naveednajam** [comment]
Yes, you are right, it's an adguard list, these list have a section which contains the host/domains to be blocked rather the full path for ads or scripts. I just take that part and add in to host list. but I guess there is still room for improvement on filtering
